warehouse/templates/email/password-reset.body.txt

```diff
@@ -0,0 +1,9 @@
+Someone, perhaps you, has made a password reset request for your PyPI account
+'{{ username }}'.
+
+If you wish to proceed with this request, please follow the link below to reset
+your password:
+
+  {{ request.route_url('accounts.reset-password', _query={'otk': otk}) }}
+
+If you did not make this request, you can simply ignore this email.
```

Comment:
Or respond to let the administrators know?

Reply:
@ewdurbin what email would we use to contact admins?  Or maybe we should link to the security policy instead?

Reply:
that's a really good point a link to the security policy and a note saying a response is OK as well would be solid.

the email will be delivered from an address that _is_ repayable and will go to PyPI Admins though.